var/spack/repos/builtin/packages/ecflow/package.py

```diff
@@ -27,11 +27,19 @@ class Ecflow(CMakePackage):
     version('4.12.0', sha256='566b797e8d78e3eb93946b923ef540ac61f50d4a17c9203d263c4fd5c39ab1d1')
     version('4.11.1', sha256='b3bcc1255939f87b9ba18d802940e08c0cf6379ca6aeec1fef7bd169b0085d6c')
 
+    variant('ssl', default=True,
```

Comment:
What is the ssl variant for?

Reply:
It enables remote connections between servers and clients via SSL - not needed in CI, it works on Orion, Discover and my Mac locally. We can fix the CI system later if needed.